getting-started/basicapp/README.md

```diff
@@ -0,0 +1,362 @@
+# Building, Testing, and Optimizing Code on the Google Cloud
```

Comment:
Would you mind removing references to "basic"? This integrates several different products and might not be straightforward for a newcomers. :)

Reply:
Changed the app name to devflowapp and removed references to 'basic'. Also removed a number of parts that made the example more complex, including kubernetes, Trace, and Profiler integrations.